### PR TITLE
Operation: Funky Nostaligia. Changes charliestation medkit icon, and adds a new heirloom item for MDs.

### PR DESCRIPTION
**Pull request by trollbreeder**
## About The Pull Request

1. Changes charliestation ancient medkits (those with bruise packs and ointment) to use the old medkit icon.
2. Adds a new family heirloom to MDs, PMs and CMOs: an old, empty medkit. It's empty because of assistants who kept breaking into medbay storage to heal themselves. This also removes the bodybag heirloom from MDs, CMOs and PMs in place of this medkit.

## Why It's Good For The Game

1. When i think of charliestation, i think of so old that the designs are from 2553. These ancient medkits use the normal medkit icon and that was a lost opportunity to me, so that is now rectified.
2. I would've expected more from the heirloom of MDs, who had to cope with a lot of changes, but apparantly not, just a body bag and a stethoscope. That's now rectified and MDs can now bring a piece of history along with them.

## Changelog
:cl:.
add: Nanotrasen has asked to return all first-aid kits produced before 2559 as they've been recalled due to potential risk of lead acetate poisoning.
add: Medical doctors have been seen emotionally attached to these outdated first-aid kits.
del: Nanotrasen reports fewer cases of necrophillia, as medical doctors no longer feel any emotions towards body bags.
imageadd: Ancient medkits are now properly ancient in design.
/:cl:

### CODE REVIEW

**Comment by trollbreeder** [comment]
god fucking damnit

**Comment by trollbreeder** [comment]
wait what the fuck did i just do

**Comment by trollbreeder** [comment]
okay that should've fixed the changelog escapade

**Comment by Fikou** [comment]
nostaligia

**Comment by ArcaneMusic** [comment]
The lead acetate lore _**DEEPENS**_

File: code/game/objects/items/storage/firstaid.dm [hunk 1]
```diff
@@ -114,7 +114,7 @@
 	generate_items_inside(items_inside,src)
 
 /obj/item/storage/firstaid/ancient
-	icon_state = "firstaid"
+	icon_state = "oldifrstaid" // yes this is the icon state its typo'd i know stop asking
```
Comment (Rohesie) [inline code/game/objects/items/storage/firstaid.dm]: Just fix it?

Comment (trollbreeder) [reply code/game/objects/items/storage/firstaid.dm]: i wanted to avoid conflicts but sure

Comment (trollbreeder) [reply code/game/objects/items/storage/firstaid.dm]: See what i mean rohesie?

Comment (Rohesie) [reply code/game/objects/items/storage/firstaid.dm]: Merge master into your branch first. After you've done that, rename it.
And done, won't be conflicts.

**Comment by trollbreeder** [comment]
shit shit shit

**Comment by trollbreeder** [comment]
rohesie this did NOT help

**Comment by trollbreeder** [comment]
don't merge while i fix this, hold on

**Comment by trollbreeder** [comment]
okay i think i truly do NOT understand github and i will probably need to make a new PR now because i went too far off course

File: code/game/objects/items/storage/firstaid.dm (hunk 1 above)
Comment (trollbreeder) [reply code/game/objects/items/storage/firstaid.dm]: I can't follow basic instructions and everything is fucked now

**Comment by cacogen** [comment]
rip

**Comment by trollbreeder** [comment]
@cacogen #50210

**Pull request closed without merging**